src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter bufferedWriter =
+                     new BufferedWriter(new FileWriter(toFileName))) {
+
+            StringBuilder stringBuilder = new StringBuilder();
+            String line = bufferedReader.readLine();
+            int supplySum = 0;
+            int buySum = 0;
+            while (line != null) {
+                stringBuilder.append(line).append(System.lineSeparator());
+                String[] parts = line.split(",");
+                int extractedNumber = Integer.parseInt(parts[1]);
+                if (parts[0].equals("supply")) {
+                    supplySum += extractedNumber;
+                } else if (parts[0].equals("buy")) {
```

Comment:
avoid using magic numbers, make it constants with meaningful names
```suggestion
                if (parts[0].equals("supply")) {
                    supplySum += extractedNumber;
                } else if (parts[0].equals("buy")) {
```